controllers/comments-controller.js

```diff
@@ -25,3 +26,14 @@ exports.postCommentToArticleId = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.deleteCommentById = async (req, res, next) => {
+  try {
+    const id = req.params.comment_id;
+    await checkResourceExists("comments", "comment_id", [id]);
```

Comment:
Shouldn't need to check existence of the comment here - you can take a look at the rows property on the result that comes back from the database - if it's an empty array, you already have your answer